compiler/pipes/gen-tree-postprocess.cpp

```diff
@@ -230,13 +232,86 @@ VertexPtr GenTreePostprocessPass::on_enter_vertex(VertexPtr root) {
 }
 
 VertexPtr GenTreePostprocessPass::on_exit_vertex(VertexPtr root) {
+  if (auto as_instance_prop = root.try_as<op_instance_prop>(); as_instance_prop && as_instance_prop->is_null_safe) {
+    if (as_instance_prop->instance().try_as<op_func_call>() || as_instance_prop->instance().try_as<op_seq_rval>() || 
```

Comment:
This condition looks complicated and it may require a comment.
It also looks like it's not complete, it covers some of the cases but not the others.
To make it more extensible, maybe move it to a method that would report whether the transformation inside this if body should happen?

Reply:
I simplified it.
I tried not to create temporary variable if it was possible, in vain. Now it looks more general and obvious.